src/couchapps/WMStats/_attachments/index.html

```diff
@@ -4,6 +4,9 @@
   <head>
     <title>WMStats</title>
     <style type="text/css" title="currentStyle">
+        @import "css/logdb.css";
+        @import "css/kube.min.css";
```

Comment:
Seangchan,
now after some sleep, after I sent you earlier kube.min.css in email
attachment I also want to let you know that it is part of WMCore ReqMgr
interface:
https://github.com/dmwm/WMCore/blob/master/src/html/ReqMgr/style/kube.min.css

so we do have but it would be nice to organize code that we can share it across
projects. What do you think? Where to place it and how to use it in both
WMStats and reqmgr2 interfaces?
Valentin.

On  0, ticoann <notifications@github.com> wrote:
> > @@ -4,6 +4,9 @@
> >    <head>
> >      <title>WMStats</title>
> >      <style type="text/css" title="currentStyle">
> > +        @import "css/logdb.css";
> > +        @import "css/kube.min.css";
> 
> Hi Valentin, This css file is missing from the head or patch? could you update that one?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6016/files#r33416760